StarVMC/xgeometry/xgeometry.age

```diff
@@ -1,4625 +0,0 @@
-* $Id: xgeometry.age,v 1.43 2021/02/25 15:48:34 jwebb Exp $
```

Comment:
I assume this deletion is a mistake.  xgeometry.age _steers the construction of the geometry for starsim_.  Please restore.